packages/firestore/src/api/database.ts

```diff
@@ -192,6 +192,37 @@ class FirestoreSettings {
       'timestampsInSnapshots',
       settings.timestampsInSnapshots
     );
+
+    // Nobody should set timestampsInSnapshots anymore, but the error depends on
+    // whether they set it to true or false...
+    if (settings.timestampsInSnapshots === true) {
+      log.error(`
+  The timestampsInSnapshots setting now defaults to true and you no
```

Comment:
Ideally we could mark the API itself deprecated and avoid complaining about this at run-time. People were not happy with us for clogging their logs when we added this the first time.

If there's no such mechanism in typescript then this is fine, but in iOS and Java where we have `NS_DEPRECATED` and `@Deprecated` I'd prefer use those and avoid logging in the case where a user has previously opted into the future behavior.

Reply:
Sounds good.